.codespellrc

```diff
@@ -0,0 +1,3 @@
+[codespell]
+exclude-file = .codespellx
+ignore-words-list = "FPT,FTP,fpt,ftp"
```

Comment:
```suggestion
ignore-words-list = "FPT,FTP,fpt,ftp,SoM"
```

```bash
codespell................................................................Failed
- hook id: codespell
- exit code: 65

docs/supported-targets/radxa-cm3.md:179: SoM ==> some
```

Reply:
@m-iwanicki, you should adhere to our internal review process. Actually we could publish it as a part of Zarhus docs.

Reply:
I see it has been fixed here 9e62b6d06b3d743c554288354cec768d748d96af.

Reply:
![image](https://github.com/user-attachments/assets/2df96f08-1760-4e3c-b605-899b7bec3613)
This button automatically commits your suggestion and closes issue